pkg/controller/command/verifiable/command_test.go

```diff
@@ -390,3 +390,60 @@ func TestGetCredentials(t *testing.T) {
 		require.Equal(t, 1, len(response.Result))
 	})
 }
+
+func TestGeneratePresentation(t *testing.T) {
+	t.Run("test generate presentation - success", func(t *testing.T) {
+		cmd, err := New(&mockprovider.Provider{
```

Comment:
Next four lines can be outside of t.Run, each test sets it up the same way.

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/f5ef9747bc23d5292e75bd2afabead5d2314681f..d8e1c2fd1338d615a9aeb8420119c409e7097a44#diff-3f7b4a4cbe8bde6826bd45d4273e5248R396).